### ISSUE TITLE
Add source code

### ISSUE THREAD

**Issue by AnimeKato** [boomerangos/Cracked-Launcher#2]
Add the source code to the [main repo](https://github.com/boomerangos/Cracked-Launcher).

**Comment by boomerangos** [comment]
I will link the original jar

**Comment by Botspot** [comment]
> I will link the original jar

@boomerangos It doesn't seem that hard to do. What's the holdup?

**Comment by AnimeKato** [comment]
> > I will link the original jar
> 
> @boomerangos It doesn't seem that hard to do. What's the holdup?

i'm not sure he knows how to work with git very well

**Comment by AnimeKato** [comment]
> I will link the original jar

no, the compiled `*.jar` files go in [releases](https://github.com/boomerangos/Cracked-Launcher/releases/), the source code goes in [code](https://github.com/boomerangos/Cracked-Launcher/).
